setup.py

```diff
@@ -31,7 +31,7 @@ def read_file(filename, encoding='utf8'):
 # specified here to avoid masking the more specific request requirements in
 # acme. See https://github.com/pypa/pip/issues/988 for more info.
 install_requires = [
-    'acme>=0.26.0',
+    'acme>=0.29.0',
```

Comment:
Actually sorry, the real correct way to fix this is to change it to 0.29.0.dev0, and the dev0 will get automatically removed at release.
```suggestion
    'acme>=0.29.0.dev0',
```